.github/ISSUE_TEMPLATE/migration-issue.md

```diff
@@ -0,0 +1,43 @@
+---
+name: "Dependabot Preview migration issue"
+about: Having issues migrating from Dependabaot Preview? Here's the place to get help!
+title: ''
+labels: 'E: preview-migration'
+assignees: '@dependabot/preview-migration-reviewers'
```

Comment:
See if I can create this team

Reply:
@dependabot/preview-migration-reviewers hi everyone 🎉